pkg/util/logger_windows_test.go

```diff
@@ -0,0 +1,28 @@
+package util
+
+import (
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/weaveworks/common/logging"
+
+	"github.com/weaveworks/common/server"
+
+	"github.com/go-kit/kit/log/level"
+)
+
+func TestWindowsLogger(t *testing.T) {
+	//wl, _ := NewWindowsLogger(nil)
+	//level.Info(wl).Log("msg", "failed to update logger", "err")
+	wl := NewWindowsEventLogger(&server.Config{
+		LogLevel: logging.Level{
+			Logrus: logrus.InfoLevel,
+			Gokit:  level.AllowAll(),
+		},
+	})
+	level.Debug(wl).Log("msg", "debug")
+	level.Info(wl).Log("msg", "info")
+	level.Warn(wl).Log("msg", "warn")
+	level.Error(wl).Log("msg", "error")
```

Comment:
This doesn't seem to be testing a lot here. We can probably do two different kinds of tests:

1. Make sure there's an error returned by `Log` when the level is too high 
2. Make sure there's no error while logging messages that should show up in the event log

Reply:
Agreed on the tests not having value.